code/modules/minimap/minimap.dm

```diff
@@ -0,0 +1,34 @@
+/obj/minimap
+	name = "Station Map"
+	layer = TURF_LAYER
+	anchored = TRUE
+	var/datum/minimap/z_level/map
+
+	New()
+		. = ..()
+		map = new /datum/minimap/z_level
```

Comment:
This will mean the map will be completely re-rendered for each new map object, not a big deal if there's only one or two of them, but might want to look into caching it if you want to use them outside of that.